docs/relay/relay.md

```diff
@@ -64,96 +83,70 @@ if (!AuthenticationService.Instance.IsSignedIn)
-/// <returns>A Task returning the needed hosting data</returns>
-public static async Task<RelayHostData> HostGame(int maxConn)
-{
-    //Initialize the Unity Services engine
-    await UnityServices.InitializeAsync();
-    //Always autheticate your users beforehand
-    if (!AuthenticationService.Instance.IsSignedIn)
-    {
-        //If not already logged, log the user in
-        await AuthenticationService.Instance.SignInAnonymouslyAsync();
-    }
+```csharp
+//Ask Unity Services to join a Relay allocation based on our join code
+JoinAllocation allocation = await Unity.Services.Relay.RelayService.Instance.JoinAllocationAsync(joinCode);
+```
 
-    //Ask Unity Services to allocate a Relay server
-    Allocation allocation = await Unity.Services.Relay.RelayService.Instance.CreateAllocationAsync(maxConn);
+## How do I pass allocation data to my transport component?
 
-    //Populate the hosting data
-    RelayHostData data = new RelayHostData
-    {
-        // WARNING allocation.RelayServer is deprecated
-        IPv4Address = allocation.RelayServer.IpV4
-        Port = (ushort) allocation.RelayServer.Port,
+Now that you have an allocation (either by joining or hosting), you need to make all traffic coming from Netcode go through the Relay. To achieve that, you need to pass allocation parameters to your transport. For now, only the `UnityTransport` supports the Relay protocol. To pass allocation parameters to it, you first need to retrieve it from your `NetworkManager` like so:
 
-        AllocationID = allocation.AllocationId,
-        AllocationIDBytes = allocation.AllocationIdBytes,
-        ConnectionData = allocation.ConnectionData,
-        Key = allocation.Key,
-    };
+```csharp
+//Retrieve the Unity transport used by the NetworkManager
+UnityTransport transport = NetworkManager.Singleton.gameObject.GetComponent<UnityTransport>();
+```
 
-    //Retrieve the Relay join code for our clients to join our party
-    data.JoinCode = await Unity.Services.Relay.RelayService.Instance.GetJoinCodeAsync(data.AllocationID);
+You can now call the method `SetRelayServerData` on the retrieved transport by passing the allocation data that you retrieved, as well as the connection type (here set to [dtls](https://docs.unity.com/relay/en/manual/dtls-encryption)). For example:
 
-    return data;
-}
+```csharp
+transport.SetRelayServerData(new RelayServerData(allocation, connectionType:"dtls"));
 ```
 
-## How do I join an existing allocation?
+Your transport is now configured. You can call `StartClient`, `StartHost` or `StartServer` and use the Netcode library as usual.
 
-The host of your game created a Relay allocation, and your client has received its [join code](https://docs.unity.com/relay/en/manual/join-codes). You now need to [request all the allocation parameters](https://docs.unity.com/relay/manual/connection-flow#4) from the join code to join the game. To do that, call the `JoinAllocationAsync` method with your join code like so:
+# Code Snippets
```

Comment:
I applied all the other suggestion, but I am not sure about this one because I think it can be misinterpreted
Maybe "Code to connect via Relay" would be more appropriate?